### PR TITLE
[REG-326] Expose Mineflayer Bot from RGBot

### PR DESCRIPTION
**Pull request by abeizer**
* Made `bot` variable in RGBot private -> bumped version to 2.0.0
* Added method `mineflayer()` that returns the mineflayer Bot
* Added method `on` that invokes Bot.on
* Updated README to reflect that an RGBot instance will be passed to `configureBot`, so players won't need to instantiate their own RGBot.

### CODE REVIEW

**Review by RG-nAmKcAz — APPROVED** [review]
LGTM

**Review by RG-nAmKcAz — CHANGES_REQUESTED** [review]
Actually.. question.. where is on() in RGBot defined ?

**Comment by abeizer** [comment]
> Actually.. question.. where is on() in RGBot defined ?

In a different project I was using to test changes... Added now

**Review by RG-nAmKcAz — APPROVED** [review]
LGTM

Don't forget to Squash/Merge as this is a new repository without everything setup

**Review by vontell — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -1,8 +1,12 @@
 {
   "name": "rg-bot",
-  "version": "1.0.0",
+  "version": "2.0.0",
```
[the remark this answers is not in the document]
Comment (vontell) [reply package.json:3]: Sure thing, I'd prefer if we make this 1.0.1

[the remark this answers is not in the document]
Comment (abeizer) [reply package.json]: Changed to 1.0.1 instead

**Review by vontell — APPROVED** [review]

**Pull request merged**